classes/uuu_bootloader_tag.bbclass

```diff
@@ -4,6 +4,7 @@
 do_deploy:append() {
     if [ "${UUU_BOOTLOADER}" != "" ]; then
         cp ${DEPLOYDIR}/${UUU_BOOTLOADER}   ${DEPLOYDIR}/${UUU_BOOTLOADER_TAGGED}
+        cp ${DEPLOYDIR}/${UUU_BOOTLOADER}   ${DEPLOYDIR}/${UUU_BOOTLOADER_UNTAGGED}
```

Comment:
Hi @hiagofranco, this bbclass is used by u-boot-imx recipe as well. Can you please adjust it for the new design?

Reply:
Sure, PR created: https://github.com/Freescale/meta-freescale/pull/1763